.travis.yml

```diff
@@ -11,13 +11,6 @@ before_script:
 script:
 - python3 manage.py test
 deploy:
-  provider: heroku
-  api_key:
-    secure: TmigMa2OGKyvftBtg9g2OpKdi83Ebg8O0i+v32nktt3/HIBD9d55BxtFYA6S4pRaFk6DtV/iIWtMLPn97DEEneIkYiNeTiDPoeuVnyhXqxJ2I4iin33vg9hDVArQxDY0BK9/vCBNYb3YR7HQjGRdNcx55XGsGlRyBwqSIWZv8W6NnPqozejQJjuSWsM3yb9a9x0WMVluukJEsR2oBWgXKm5LJqN4ofar4T2W5WEyaNZ7smZFOLZCKz9uVgrn7n2S8BMP0lBnPCgQeZK0vDAz08SBcci8550S6Nc+hW8D1DLK0qIpdky/4RAj+xAzNcpjAHTMKjDlZZPgC/F5LxFpLNFctWbeAL/QzAHa0wdKpzntScDaTlpYNMbphWBuxmW8/4VZA9SlnD+XLzj8XYPc/4ixBJLl5jc2Gemxs3mTRd6XMhYbZWlLI1XtdE+emfmY3kGtnStFRTZnEUzwRrSqvJ+PKk3I4QBlLh8tpcPuYTSqhtKlVR/vUxm2nN9ZXGI755GapqAjftTIjdpUlXMwa/juL4q2uMx6AmrQMXrLcjSDKttUb+Mafd7peDgTqfLt8kmVeF9EP3FoTIO7R5tNTi4FOVJegU0TJT0SCP3H98tFIcI8tV/x2LnEd2vIdj3MuVKs6iyUnA8xzc6bGOxo2G/H5G5WNaMdRTrOv/aJmik=
-  app: language-learner-be
-  on:
-    repo: nancylee713/AILanguageTutor_BE
-    branch: master
```

Comment:
We are no longer deploying this, correct? We'll just go from staging?

Reply:
correct